From ba92445d022f2533fcd39bad116c0d704bbc63ea Mon Sep 17 00:00:00 2001
From: Huibin Hong <huibin.hong@rock-chips.com>
Date: Fri, 27 Mar 2020 17:23:39 +0800
Subject: [PATCH] Revert "soc: rockchip: rk_fiq_debugger: clear time out
 interrupt but no data ready"

This reverts commit 08cb05ec40e08e142093fa45afda54ee6e91d7dc.

Change-Id: Ic8ffa074edcf5720cd1db55b9bf3db83db1b08cd
Signed-off-by: Huibin Hong <huibin.hong@rock-chips.com>
---
 drivers/soc/rockchip/rk_fiq_debugger.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/soc/rockchip/rk_fiq_debugger.c b/drivers/soc/rockchip/rk_fiq_debugger.c
index 5f0b2891660d..2e1f320ccd1d 100755
--- a/drivers/soc/rockchip/rk_fiq_debugger.c
+++ b/drivers/soc/rockchip/rk_fiq_debugger.c
@@ -176,8 +176,6 @@ static int debug_getc(struct platform_device *pdev)
 		} else {
 			return temp;
 		}
-	} else {
-		rk_fiq_read(t, UART_RX);
 	}
 
 	return FIQ_DEBUGGER_NO_CHAR;
-- 
2.35.3

